### ISSUE TITLE
BREAKING: `grape` color is no more

### ISSUE THREAD

**Issue by obedm503** [argyleink/open-props#362]
Version `1.5.0` renamed the `grape` color to `purple`. This is a breaking change. Please bring back `grape` as a duplicate of `purple` or release version 2 to denote the breaking change.

**Comment by obedm503** [comment]
Also, new npm releases don't show up on the [releases](https://github.com/argyleink/open-props/releases) page.

**Comment by argyleink** [comment]
you're right, it was breaking, and a major version should have been changed. i also need to setup [git release publishing](#363). let me think on this some more though. thanks for creating the issue.
